aw_watcher_window/main.py

```diff
@@ -71,11 +70,7 @@ def heartbeat_loop(client, bucket_id, poll_time, strategy, exclude_title=False):
             current_window = get_current_window(strategy)
             logger.debug(current_window)
         except Exception as e:
-            logger.error(
-                "Exception thrown while trying to get active window: {}".format(e)
-            )
-            traceback.print_exc()
-            current_window = {"app": "unknown", "title": "unknown"}
+            logger.exception("Exception thrown while trying to get active window")
```

Comment:
This will incorrectly send the last `current_window` as the new `current_window`. Needs a `sleep()` and `continue` to prevent it.

Reply:
🤦 fixed!